Header header

float32 dist_to_camera
float32 angle_to_camera

MocapEvalObservation[] obs
